components/reddit/sources/new-link-on-a-subreddit/new-link-on-a-subreddit.js

```diff
@@ -0,0 +1,84 @@
+const reddit = require("../../reddit.app.js");
+const get = require("lodash.get");
+
+module.exports = {
+  key: "new-link-on-a-subreddit",
+  name: "New Link on a subreddit",
+  description: "Emmits a new link submitted to a subreddit",
+  version: "0.0.2",
+  props: {
+    reddit,
+    timer: {
+      label: "Polling schedule",
+      description:
+        "Pipedream poll for new links and emit accordingly with this schedule.",
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 10, // by default, run every 10 minute
+      },
+    },
+    subreddit: {
+      type: "string",
+      label: "Subreddit",
+      description: "Enter subreddit name to listen for new links.",
+      default: "redditdev",
+    },
+    db: "$.service.db",
+  },
+  hooks: {
+    async deploy() {
+      // Emmits all existing events for the first time.
+      var after = null;
```

Comment:
var -> let

Reply:
Interesting. I checked javascript details on diff between var and let and I learned that I was always using var to have the scope of the block it is writing, and thus I've been trying to use let all this time.